tpgtools/serialization.go

```diff
@@ -23,6 +23,8 @@ import (
 
 	assuredworkloads "github.com/GoogleCloudPlatform/declarative-resource-client-library/services/google/assuredworkloads"
 	assuredworkloadsBeta "github.com/GoogleCloudPlatform/declarative-resource-client-library/services/google/assuredworkloads/beta"
+	bigqueryreservation "github.com/GoogleCloudPlatform/declarative-resource-client-library/services/google/bigqueryreservation"
```

Comment:
Bigqueryreservation changes should not be relevant, the samples are handwritten. It is my fault I did not generate the serialization changes anyway when I merged.